subsys/profiler/profiler_nordic.c

```diff
@@ -56,10 +56,26 @@ static K_THREAD_STACK_DEFINE(profiler_nordic_stack,
 			     CONFIG_PROFILER_NORDIC_STACK_SIZE);
 static struct k_thread profiler_nordic_thread;
 
-static void send_system_description(void)
+static void send_info_data(const char *data, size_t data_len)
 {
 	size_t num_bytes_send;
 
+	num_bytes_send = SEGGER_RTT_WriteNoLock(
+				  CONFIG_PROFILER_NORDIC_RTT_CHANNEL_INFO,
+				  data, data_len);
+
+	while (num_bytes_send == 0) {
```

Comment:
It seems that if host disconnects it will block. Maybe we should have some failsafe and return error?

Reply:
By default this thread has low priority and its preemptive, so it would not block the system.
Still I added the max retry count.